mesheryctl/pkg/utils/testing.go

```diff
@@ -167,6 +167,10 @@ func SetupLogrusGrabTesting(t *testing.T) *bytes.Buffer {
 // setup custom context with SetupCustomContextEnv
 func SetupCustomContextEnv(t *testing.T, pathToContext string) {
 	viper.Reset()
+	ViperCompose = viper.New()
+	ViperDocker = viper.New()
+	ViperK8s = viper.New()
+
```

Comment:
We want to reset all the instances of Viper in between tests. For example, in ChangePlatform that is called by TestChangePlatform, the platform in ViperK8s is changed to "testplatform" under context "gke". In TestChangeConfigEndpoint, we are changing the endpoint under "kubernetes" context using the same ViperK8s variable. So, Viper needs to re-initialize itself and the config it stores in between tests, so that the previous values aren't persisted between tests.